static_src/components/disclaimer.jsx

```diff
@@ -15,20 +15,52 @@ export default class Disclaimer extends React.Component {
 
   render() {
     let flagImg;
+    let dotGovIconImg;
+    let httpsIconImg;
+    const dotGovIcon = require('cloudgov-style/img/icon-dot-gov.svg');
+    const dotGovAlt = 'Dot gov';
+    dotGovIconImg = <img className={ this.styler('usa-banner-icon','usa-media_block-img') } src={ dotGovAlt } src={ dotGovIcon }</img>;
+    const httpsIcon = require('cloudgov-style/img/icon-https.svg');
+    const httpsAlt = 'Https';
+    httpsIconImg = <img className={ this.styler('usa-banner-icon','usa-media_block-img') } src={ httpsAlt } src={ httpsIcon }</img>;
```

Comment:
As above, I don't think we need to store this tag in a variable.

I think the first `src` attribute should be `alt`.

Also, image tags are self closing, you can omit the `</img>`

Reply:
Sounds good. I was following the pattern on the previous template. I'll update accordingly.